.builds/openbsd.yml

```diff
@@ -38,7 +38,7 @@ tasks:
         # https://todo.sr.ht/~sircmpwn/builds.sr.ht/274
         ln -s ${HOME}/rizin-testbins test/bins
         # Running the unit tests
-        MALLOC_OPTIONS=S ninja -C build test
+        ninja -C build test
```

Comment:
Better not disable the `MALLOC_OPTIONS` imo. They are nice for detecting bugs that are invisible on other platforms.

Reply:
Ok ddb7d56fe1fc1dd2e8ef70085945858dbd51e18d but there won't be any speedup